.gitignore

```diff
@@ -7,3 +7,5 @@
 **/*.xcodeproj/*
 
 user.bazelrc
+
+.vscode
```

Comment:
Added some project specific settings I wished to ignore

Reply:
Can you submit that as it's own PR? Thanks!